src/js/utils/__tests__/ReducerUtil-test.js

```diff
@@ -0,0 +1,163 @@
+const ReducerUtil = require('../ReducerUtil');
+
+const idReducer = function (state = '', action) {
+  if (action.path.join('') === 'id') {
```

Comment:
Not sure understand the idea behind this part, having an array so you need to do `action.path.join('')`. Could you explain a use case where this is necessary/helpful?